modules/shared/components/atoms/DropDown/DropDown.module.css

```diff
@@ -1,13 +1,13 @@
 .menu-icon-container {
```

Comment:
I liked how you positioned the dropdown without breaking the responsive of the page